clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -529,6 +531,9 @@ llvm::Type *CodeGenTypes::ConvertType(QualType T) {
       }
     case BuiltinType::SveCount:
       return llvm::TargetExtType::get(getLLVMContext(), "aarch64.svcount");
+    case BuiltinType::MFloat8:
```

Comment:
What is the  difference between this change and before.
It looks like both are generation a vector of: <Info.EC x i8>. But in this case Info.EC  is replaced by 1.

Do we need case BuiltinType::MFloat8? Why the changes are not enough for the scalar type?

Reply:
How else would you expect this function to work when you call it with an argument that is `__mfp8` ?

Reply:
> What is the difference between this change and before.
Ah, and the difference is that `__mfp8` is no longer an `AARCH64_VECTOR_TYPE`